src/index.css

```diff
@@ -0,0 +1,198 @@
+  font-family: "Quicksand";
+  font-weight: 500;
+}
+
+.app {
+  width: 100%;
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+}
+
+h1 {
+  color: #b91717;
+  text-align: center;
+  background-color: bisque;
+  font-family: "Sedgwick";
+  font-size: 8rem;
+  text-transform: uppercase;
+  font-weight: 400;
+  word-spacing: 2.5rem;
+  letter-spacing: -0.4rem;
+  padding: 1.8rem 0;
+}
+
+.add-form {
+  background-color: #800000;
+  padding: rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
+}
+
+h3 {
+  color: #fff;
+  margin-right: 2rem;
+  font-size: 1.5rem;
+}
+
+button,
+select,
+input {
+  outline: 0;
+  background-color: #fff;
+  color: black;
+  font-family: inherit;
+  border: none;
+  border-radius: 6rem;
+  padding: 0.8rem 2rem;
+  font-weight: 700;
+  font-size: 1rem;
+  cursor: pointer;
+  margin-block: 1rem;
+}
+
+.add-form button {
+  color: #fff;
+  text-transform: uppercase;
+  background-color: #b91717;
+}
+
+.add-form button:hover {
+  background-color: red;
+}
+
+.list {
+  background-color: #b91717;
+  color: #ffebb3;
+  padding: 2.5rem 0;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  gap: 2rem;
+  align-items: center;
+}
+
+.actions button,
+.list select {
+  text-transform: uppercase;
+  padding: 0.5rem 1.5rem;
+  font-size: 0.8rem;
+  font-weight: 700;
+  margin: 0 0.5rem;
+}
+
+.actions button:hover,
+.list select:hover {
+  transform: scale(1.04);
+  transition: 0.2s ease-in;
+}
+
+.list ul {
+  list-style: none;
+  width: 80%;
+  overflow: hidden;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  gap: 0.6rem;
+  justify-content: center;
+  align-content: start;
+}
+
+.list li {
+  display: flex;
+  align-items: center;
+  gap: 0.8rem;
+}
+
+input[type="checkbox"] {
+  height: 1rem;
+  width: 1rem;
+  accent-color: #fff;
+}
+
+.list li button {
+  cursor: pointer;
+  background: none;
+  border: none;
+  font-size: 1.2rem;
+  padding: 0.5rem;
+  transform: translateY(2px);
+}
+
+.stats {
+  background-color: bisque;
+  text-align: center;
+  font-weight: 700;
+  padding: 2rem 0;
+}
+
+@media screen and (max-width: 768px) {
+  .list ul {
+    margin-inline-start: 10rem;
+    width: 90%;
+  }
+
+  body {
+    font-size: 1rem;
+  }
+
+  h1 {
+    font-size: 4rem;
+  }
+
+  button,
+  select,
+  input {
+    border-radius: 6rem;
+    padding: 0.4rem 1rem;
+    font-weight: 700;
+    font-size: 0.6rem;
+    cursor: pointer;
+    margin-block: 0.5rem;
+  }
+
+  .actions {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .actions button {
+    padding: 0.3rem 0.8rem;
+    font-size: 0.6rem;
+    margin-block: 2rem;
+  }
+
+  .add-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 0.5rem;
+  }
+}
+
+@media screen and (max-width: 568px) {
+  .list ul {
+    width: 80%;
+    margin-inline-start: 20rem;
```

Comment:
Pe ecran de telefon, conținutul îți iese din ecran.

Reply:
Am rezolvat, pusesem niste margini aiurea in <ul> in @media, le-am scos.